birdhouse/components/cowbird/default.env

```diff
@@ -50,6 +50,12 @@ export COWBIRD_LOG_LEVEL=INFO
 # Subdirectory of DATA_PERSIST_SHARED_ROOT containing the user workspaces used by Cowbird
 export USER_WORKSPACES="user_workspaces"
 
+THIS_FILE="`realpath "$0"`"
```

Comment:
Do not add this `THIS_FILE` var name, which is too generic and will clash with other scripts.

The info you want is already capture by the var `COMPOSE_DIR`, set here

https://github.com/bird-house/birdhouse-deploy/blob/6584dff1574df40e897d15dc4499b63e54e82f2a/birdhouse/pavics-compose.sh#L44-L50

Reply:
This will be much cleaner, I hadn't successfully found that variable. I updated it.